invenio_queues/__init__.py

```diff
@@ -6,7 +6,126 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio module for collecting statistics."""
+
+r"""Invenio module for managing queues.
+
+This guide will show you how to get started with Invenio-Queues. It assumes
+that you already have knowledge of Flask applications and Invenio modules.
+
+It will then explain key topics and concepts of this module.
+
+Getting started
+---------------
+
+You will learn how to register queues and to interact with it. To begin with,
+you need to setup your virtual environment and install this module.
+
+You need an application to work with, that can be created with the following
+commands in a Python shell:
+
+>>> from flask import current_app
+
+You can then initialize the module:
+
+>>> from invenio_queues.ext import InvenioQueues
+>>> ext_queues = InvenioQueues(app)
+
+In our example, we are using RabbitMQ as a broker, which can be configured
+as follow:
+
+>>> current_app.config['QUEUES_BROKER_URL'] = 'amqp://localhost:5672'
+
+Register queues
+^^^^^^^^^^^^^^^
+
+To register queues, you need to start by creating an exchange for the queues:
+
+.. code-block:: python
+
+   from kombu import Exchange
+
+   default_exchange = Exchange(
+       'example',
+       type='direct',
+       delivery_mode='transient',  # in-memory queue
+   )
+
+You can now configure the queues as followed:
+
+.. code-block:: python
+
+   from invenio_queues.proxies import current_queues
+   from invenio_queues.queue import Queue
+
+   current_queues.queues = dict()
```

Comment:
This might a convenient way to test locally while writing this documentation, but should not be documented as a standard way to define queues. It should only be documented to use the `invenio_queues.queues` entrypoint for defining queues.